README.md

```diff
@@ -41,7 +41,7 @@ This project supports the following guest operating systems:
 | Oracle Linux                 | 8         |
 | CentOS Stream                | 9         |
 | CentOS Stream                | 8         |
-| Fedora Server                | 39        |
+| Fedora Server                | 41        |
```

Comment:
Also here: https://github.com/vmware-samples/packer-examples-for-vsphere/blob/fb37d5e8488c5f253461c8328e7cad32202d76a5/docs/index.md?plain=1#L37